bazarr/subtitles/upgrade.py

```diff
@@ -303,21 +362,66 @@ def get_upgradable_movies_subtitles():
         .subquery()
 
     minimum_timestamp, query_actions = get_queries_condition_parameters()
+    logging.debug(f"Minimum timestamp used for subtitles upgrade: {minimum_timestamp}")
+    logging.debug(f"Those actions are considered for subtitles upgrade: {query_actions}")
 
     upgradable_movies_conditions = [(TableHistoryMovie.action.in_(query_actions)),
                                     (TableHistoryMovie.timestamp > minimum_timestamp),
                                     TableHistoryMovie.score.is_not(None),
                                     (TableHistoryMovie.score < 117)]
     upgradable_movies_conditions += get_exclusion_clause('movie')
-    return select(TableHistoryMovie.id) \
-        .select_from(TableHistoryMovie) \
+    subtitles_to_upgrade = database.execute(
+        select(TableHistoryMovie.id,
+               TableHistoryMovie.video_path,
+               TableHistoryMovie.language,
+               TableHistoryMovie.upgradedFromId)
+        .select_from(TableHistoryMovie)
+        .join(TableMovies, onclause=TableHistoryMovie.radarrId == TableMovies.radarrId)
         .join(max_id_timestamp, onclause=and_(TableHistoryMovie.video_path == max_id_timestamp.c.video_path,
                                               TableHistoryMovie.language == max_id_timestamp.c.language,
-                                              max_id_timestamp.c.timestamp == TableHistoryMovie.timestamp)) \
-        .join(TableMovies, onclause=TableHistoryMovie.radarrId == TableMovies.radarrId) \
-        .where(reduce(operator.and_, upgradable_movies_conditions)) \
-        .order_by(TableHistoryMovie.timestamp.desc()) \
-        .subquery()
+                                              max_id_timestamp.c.timestamp == TableHistoryMovie.timestamp))
+        .where(reduce(operator.and_, upgradable_movies_conditions))
+        .order_by(TableHistoryMovie.timestamp.desc())) \
+        .all()
+    logging.debug(f"{len(subtitles_to_upgrade)} subtitles are candidates and we've selected the latest timestamp for "
+                  f"each of them.")
+
+    query_actions_without_upgrade = [x for x in query_actions if x != 3]
```

Comment:
Magic number 3.
I think it's the same as the earlier magic number 3, but can I be sure?!

Reply:
Yes you can, I assure you it is!